firefox-ios/Client.xcodeproj/project.pbxproj

```diff
@@ -16,11 +16,18 @@
 		047F9B3E24E1FF4000CD7DF7 /* SearchQuickLinks.swift in Sources */ = {isa = PBXBuildFile; fileRef = 047F9B3A24E1FF4000CD7DF7 /* SearchQuickLinks.swift */; };
 		047F9B4224E1FF4000CD7DF7 /* ImageButtonWithLabel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 047F9B3C24E1FF4000CD7DF7 /* ImageButtonWithLabel.swift */; };
 		0A49784A2C53E63200B1E82A /* TrackingProtectionViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0A4978492C53E63200B1E82A /* TrackingProtectionViewController.swift */; };
+		0A7693612C7DD19600103A6D /* CertificatesModelTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0A7693602C7DD19500103A6D /* CertificatesModelTests.swift */; };
 		0AC659272BF35854005C614A /* FxAWebViewModelTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0AC659262BF35854005C614A /* FxAWebViewModelTests.swift */; };
 		0AC659292BF493CE005C614A /* MockFxAWebViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0AC659282BF493CE005C614A /* MockFxAWebViewModel.swift */; };
 		0AD3EEAC2C2485A7001044E5 /* ThemedCenteredTableViewCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0AD3EEAB2C2485A7001044E5 /* ThemedCenteredTableViewCell.swift */; };
+		0AFF7F5E2C74DCA300265214 /* (null) in Sources */ = {isa = PBXBuildFile; };
+		0AFF7F602C75D3D300265214 /* (null) in Sources */ = {isa = PBXBuildFile; };
+		0AFF7F622C77432900265214 /* (null) in Sources */ = {isa = PBXBuildFile; };
```

Comment:
Hmm, these 3 lines look a bit odd. 🤔 

Each of these 3 keys (e.g. `0AFF7F622C77432900265214`) also have another `(null)` line lower in this file too (e.g. line 15090).

You may need to manually delete these lines (and the corresponding ones below), as I suspect something strange happened and they shouldn't be in the project file.

Reply:
Don't even know how I achieved that. Thank you.